STEP_Project/content.js

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
I think we should rename this file to ouselect_content.js.

Otherwise, there will be conflicts when we do it for user and group.

Reply:
File was renamed to OUListContent.js since we will be having another content script for the Select page, and will call that one OUSelectContent.js